meetings/roles.md

```diff
@@ -99,13 +99,13 @@ This time should be scheduled as close as possible after the end of the meeting.
 **During Meeting**
 
 * [ ] Note the first names of each meeting attendee.
-* [ ] At the beginning, encourage everyone to type in key points and assignments directly into the document.
+* [ ] At the beginning, encourage everyone to type in key points and assignments directly into the document or into a raw notes section at the bottom of the document. 
 * [ ] Watch closely what is being typed to ensure that all key points and assignments make it in.
 Make note to yourself of any missed information.
 
 **Within 24 Hours of Meeting**
 
-* [ ] Clean up everyone's notes plus your own into a cohesive set of key points and assignments of the meeting.
+* [ ] Clean up everyone's notes **Notes** of the notes. Use the raw notes section plus your own notes to provide a cohesive set of key points and assignments of the meeting. Do not alter or remove the raw notes. 
```

Comment:
Does that work the checkbox feature in Markdown though?
```suggestion
* [ ] Clean up everyone's **Notes** of the notes. Use the raw notes section plus your own notes to provide a cohesive set of key points and assignments of the meeting. Do not alter or remove the raw notes. 
```
 - Removed excessive looking `notes` since the sentence didn't parse for me anymore. 
 - "The Notes of the notes." Maybe this can be phrased a bit more clear. 
-  "your own notes" Would that refer to notes taken offline due to whatever reasons such as cases of [Neuland](https://www.youtube.com/watch?v=-VkLbiDAouM) (broken connection / hickups) or similar things? Maybe there's a chance to phrase that more clearly.

Reply:
> Does that work the checkbox feature in Markdown though?

Yes, still works.

Reply:
> "The Notes of the notes." Maybe this can be phrased a bit more clear.

```suggestion
* [ ] Use the raw notes section plus your own notes to provide a cohesive set of key points and assignments of the meeting.
Do not alter or remove the raw notes. 
```